server/src/routes/index.js

```diff
@@ -12,6 +13,13 @@ router.get("/sideBar", InfoCtrl.getSideBarInfo);
 
 router.get("/createGame", GameCtrl.create);
 
+// temporary to test db function
+router.post('/dbWriteTemp', newUser.userCreate);
```

Comment:
Why did you add 'Temp' to the route name?
If you intend to keep it, remove the Temp